react/README.md

```diff
@@ -450,6 +450,7 @@
     ```
 
   - Do not use underscore prefix for internal methods of a React component.
+  > Why? Underscore prefixes are sometimes used as a convention in other languages to denote privacy. But, unlike those languages, there is no native support for privacy in JavaScript, everything is public. Regardless of your intentions,  adding underscore prefixes to your properties does not actually make them private, and any property (underscore prefixed or not) should be treated as being public. See issues #1024, and #490 for a more in-depth discussion.
```

Comment:
PHP uses protected and private keywords. there's no need to use underscores

On Sat, Aug 27, 2016, 6:57 PM Jordan Harband <notifications@github.com>
wrote:

> In react/README.md
> <https://github.com/airbnb/javascript/pull/1025#discussion_r76519166>:
>
> > @@ -450,6 +450,7 @@
> >      ```
> >
> >    - Do not use underscore prefix for internal methods of a React component.
> > +  > Why? Underscore prefixes are sometimes used as a convention in other languages to denote privacy. But, unlike those languages, there is no native support for privacy in JavaScript, everything is public. Regardless of your intentions,  adding underscore prefixes to your properties does not actually make them private, and any property (underscore prefixed or not) should be treated as being public. See issues #1024, and #490 for a more in-depth discussion.
>
> there shouldn't be two spaces after "intentions,", and let's do
> "underscore-prefixed".
>
> In what other languages is a single underscore used as a convention for
> privacy? I'm only aware of it in JS. PHP (and I think Python) use double
> underscores, but that's not the convention we're addressing here.
>
> Otherwise, looks good!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/airbnb/javascript/pull/1025/files/93bf976765cbca61139c092652c6ae5782fa1955#r76519166>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABBxEFN4hRQoYnUwiOj5tqs7pQvgQFmBks5qkGxlgaJpZM4JpUov>
> .
>